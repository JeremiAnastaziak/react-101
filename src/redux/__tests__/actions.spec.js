import * as actions from '../actions';

describe('actions', () => {
  test('returns correct increment action', () => {
    expect(actions.increment()).toMatchSnapshot();
  });

  test('returns correct decrement action', () => {
    expect(actions.decrement()).toMatchSnapshot();
  });
});
