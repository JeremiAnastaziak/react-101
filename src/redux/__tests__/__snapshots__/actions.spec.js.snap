// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`actions returns correct decrement action 1`] = `
Object {
  "payload": Object {
    "amount": -1,
  },
  "type": "DECREMENT",
}
`;

exports[`actions returns correct increment action 1`] = `
Object {
  "payload": Object {
    "amount": 1,
  },
  "type": "INCREMENT",
}
`;
